modules/sc-mesh-secure-deployment/src/nats/comms_nats_controller.py

```diff
@@ -500,26 +499,35 @@ def __write_config_to_file(response: requests.Response) -> None:
         :param response: HTTP response
         :return: -
         """
-        with open("/opt/config.json", "w", encoding="utf-8") as f:
-            f.write(response.text.strip())
+        with open("/opt/config.json", "w", encoding="utf-8") as f_handle:
+            f_handle.write(response.text.strip())
 
     def upload_certificate_bundle(self) -> requests.Response:
+        """
+        Upload certificate bundle
+        :return: HTTP response
+        """
         try:
-            # TODO: Not sure what kind of bundle we should deliver
-            # so bundling just device cert and ca cert for now.
-            with open(self.__certificate_file, 'r') as cert_file:
-                certificate_content = cert_file.read()
+            certificates_dict = {}
+
+            files = glob.glob("/opt/at_birth.tar.bz2*")
 
-            with open(self.__ca, 'r') as ca_file:
-                ca_certificate_content = ca_file.read()
+            if 4 > len(files) or len(files) == 0:
```

Comment:
Not urgent now but to avoid a risk of this maybe breaking later: we do not need exactly 4 files, we need the at_birth.tar.bz2 archive itself of course but with regards to the digital signatures we technically need only one, so it would be a pity to refuse to upload if some 3rd party software update left us with only 1 or 2 out of 3 signatures available :-)

Reply:
I will remove the check.

Reply:
or change it to return fail if there is not any.

Reply:
Fixed with  "if len(files) == 0  then return fail"

Reply:
Thank you :)